### ISSUE TITLE
CVC train

### ISSUE THREAD

**Issue by lihui1998** [sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection#13]
Thank you for your reply, since we are very interested in your work, we hope to reproduce the data in the paper. Based on your previous suggestion, our issue remains unresolved. Below I describe the work we do in detail, and hope to get your suggestions. We first resized the images on the basis of the original dataset, unified to 640×471, and then we merged the contents of the FramesNeg and FramesPos folders in the training set into the Frames file. After that we trained using the datasets.py, CVC-14_test_all_Full.json, Images_train_all.txt and Images_test_all.txt you provided, since we still used the config.py file for the kaist dataset, we did the datasets.py file Modified as follows;
![2ed74a4523146f5bebd977c1b0bf062](https://user-images.githubusercontent.com/51204863/154802381-c13b81cf-731e-4aa4-8199-35af7b3f2553.png)
In addition, we also modified some details in the eval code;
![9812c5ec465b4257bb9af159445dadf](https://user-images.githubusercontent.com/51204863/154802392-1d2f6153-c662-4183-812f-f561ae23453e.png)
Although it can be trained, it can't go down during the day as described in the previous question. In addition, according to your suggestion, we only use the bounding box in the gray image for training and testing, and no longer convert the image in visible to RGB during training. , but the effect is worse. Can you give further advice on our work? Thank you so much, looking forward to your reply!

**Comment by socome** [comment]
I think you need to change the code because there are some different between MLPD and MLPD† in the process of NMS. Could you try to change the code related all of NMS such as [it](https://github.com/sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection/blob/8e583929a97b9f959634efa69174435ad0252a57/src/model.py#L568)? Thank you.
